### ISSUE TITLE
New module - Hibernate reactive (suggestion) 

### ISSUE THREAD

**Issue by aaloise** [danielprinz/vertx-udemy#2]
Hello. May I suggest you to improve the course by creating a module explaining how to use Hibernate Reactive, since it uses Vert.x under the hood?

**Comment by danielprinz** [comment]
Hello @aaloise 👋 
have you seen the Quarkus & Vert.x section? There I cover Hibernate Reactive briefly.
Or are you more looking into using Vert.x & Hibernate Reactive without Quarkus?

Kind regards, Daniel

**Comment by aaloise** [comment]
Hi @danielprinz ! Didn't notice that Hibernate Reactive was under Quarkus & Vert.x section.

But like you pointed, my proposal is something more like Vert.x & Hibernate. Just this two and no Quarkus involved

**Comment by danielprinz** [comment]
I agree, a special Section for Vert.x & Hibernate Reactive makes sense. Thanks for the suggestion.
